docs/projects/README.md

```diff
@@ -0,0 +1,453 @@
+  - Ideally PRs can be reviewed by anyone capable of reviewing in the relevant
+    parts of the codebase, not only people intimately familiar with the project
+  - This isn't always possible, but it is good to strive for it as much as we
+    can
+- Any milestones. Moments when significant or discrete chunks of work have been
+  completed, or sub-features can ship.
+- Any accessibility concerns or requirements.
+- Rollback
+  - How do we rollback this solution in the event of failure?
+  - Are there any steps that can not easily be rolled back?
+- Privacy
+  - How does this approach protect users' privacy?
+- Localization.
+  - Any translation or regional requirements?
+  - Any differing legal requirements based on user location?
+- Risks:
+  - What risks are we taking with this solution?
+  - Are there risks that once taken can’t be undone?
+- Prior art:
+  - Include links to documents and resources that you used when coming up with
+    your solution.
+  - Credit people who have contributed to the solution that you wish to
+    adknowledge.
+
+> **Note**
+>
+> We do not have any reccommendations about time estimates in this section.
+
+#### The review process
+
+Technical implementation plans will be open for a two-week review period. In
+cases where relevant contributors are AFK or otherwise unavailable, or a
+implementation plan experiences low engagement for a number of reasons,
+implementation plan deadlines will be extended by one week at a time
+as-necessary.
+
+#### Wrapping up
+
+Finally, the implementation plan is merged into the repo. Project contributors
+create GitHub issues for all of the work identified in the implementation plan.
+This process can be quite time-consiming for large projects. Generally, it makes
+sense to create issues in the order which they must be completed; this allows
+work to begin if, for some reason, there is delay in creating issues for work at
+the end of a project. It also however makes sense to priorize issues that are
+"good first issues" and "help wanted" issues which contributors might be able to
+help with.
+
+It is important to make sure dependencies are documented when creating these
+issues. Issues which depend on another issue or set of issues should be labeled
+as "blocked" and reference the blocking issue(s).
+
+### Implementation (`status: In Progress`)
+
+In the implementation phase, contributors begin work on the project. At this
+point the work has been divided into discrete tasks and ordered according to
+priority. Folks who have _not read_ the motivating documents or who even lack a
+full understanding of the project should be able to jump in and work on
+individual pieces of the project by _exclusively_ reading the information
+present in the GitHub issue or task they are being assigned.
+
+> **Warning**
+>
+> If this isn't possible, planning was insufficient, and we may want to place
+> the project in `status: On Hold` to re-adjust. It's a very important aspect of
+> the work.
```

Comment:
I think I understand the motivation behind this, but it kind of worries me because it could imply two things: either all the information from the implementation plan needs to be duplicated into the issues, in particular over-arching details that are necessary to keep in mind; or that each implementation step needs to be planned to the level of "lines of code" so that implementation is almost/exactly copy-paste kind of job. To me, something in between is ideal. It also strikes me that the current description of this issue does not provide any actionable step to remediate it other than putting a project on hold.

Is the sentiment here that we want to alert people for when planning was insufficient? Or is it guidance for issue writing? If it is guidance for issue writing, we should move it into the previous section of the document (Wrapping Up the implementation planning) so that it is alongside other guidance for issue writing. If it is guidance for people implementing, then it needs to be rephrased.

If it is guidance to try to capture when implementation planning did not sufficiently capture the requirements for an issue, then perhaps the instructions could read:

> If at any point during implementation, it is discovered that the implementation planning did not sufficiently capture the details of the project, we will put the project back into a planning mode. Someone familiar with the broader scope of the project will be assigned to dig into the discovered discrepancies and sufficiently resolve them for the project to continue in confidence. While issues ideally stand on their own (and do not require reading the full implementation plans) for some issues, complexity prevents this from being possible. If the contributor working on an issue is still unable to achieve clarity on the details of an issue, even after reading the relevant parts of the implementation plan, this would be a signal that details were not sufficiently captured during implementation.
>
> Note that this is distinct from when the implementation plan has explicitly called out that an ambiguity exists that will need to be resolved after a specific part of the implementation is complete. The point we're getting at here is that if at any point during implementation, contributors are unclear on what they are meant to implement or how they're meant to implement it _and this wasn't an expected part of the project during planning_, then something potentially big was missed during planning. In these cases, it is worth taking a step back to ensure that the full scope of the plan is defined and being followed.

This does feel like an important detail, and I'm glad that you identified this as a step where this risk could be easily identified. I just want to make sure that we capture the broader intention so that the "why" behind this is clear, otherwise it feels arbitrary and hard to understand what is necessary to follow the guidance.

Reply:
This makes a lot of sense, I'll adjust and @ you in the revised sections.